functions/send_mods.php

```diff
@@ -28,131 +32,87 @@ function slugify($text) {
   }
   return $text;
 }
-$fileName = explode("-",slugify($fileName));
-array_pop($fileName);
-$fileName = implode("-",$fileName).".jar";
 
-if(!file_exists("../mods/mods-".$fileName)) {
-	mkdir("../mods/mods-".$fileName);
+$fileNameTmp = explode("-",slugify($fileName));
+array_pop($fileNameTmp);
+$fileNameShort=implode("-",$fileNameTmp);
+$fileNameZip=$fileNameShort.".zip";
+$fileName=$fileNameShort.".jar";
+
+$fileJarInFolderLocation="../mods/mods-".$fileNameShort."/".$fileName;
+$fileZipLocation="../mods/".$fileNameZip;
+$fileInfo=array();
+
+if(!file_exists("../mods/mods-".$fileNameShort)) {
+	mkdir("../mods/mods-".$fileNameShort);
 } else {
-	echo '{"status":"error","message":"Folder mods-'.$fileName.' already exists!"}';
+	echo '{"status":"error","message":"Folder mods-'.$fileNameShort.' already exists!"}';
 	exit();
 }
 
-if(move_uploaded_file($fileTmpLoc, "../mods/mods-".$fileName."/".$fileName)){
```

Comment:
Wondering why this block was removed. If you could explain it, I cannot see why.